Source/Examples/WPF.SharpDX/DeferredShadingDemo/DeferredShadingDemo_NET40.csproj

```diff
@@ -45,9 +46,11 @@
       <HintPath>..\..\..\packages\PropertyTools.Wpf.2014.1.29.1\lib\NET40\PropertyTools.Wpf.dll</HintPath>
     </Reference>
     <Reference Include="SharpDX">
-      <HintPath>..\..\..\packages\SharpDX.2.5.0\lib\net40\SharpDX.dll</HintPath>
+      <HintPath>$(SharpDXPackageBinDir)\SharpDX.dll</HintPath>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.Windows.Forms" />

```

Comment:
These are also, annoyingly, added by the SharpDX nuget installer script. They are not required, and I'll remove them.